src/references/audio.md

```diff
@@ -90,72 +88,79 @@ This is the radius at which the sound plays at full volume in centimeters (cm).
 
 #### Falloff
 
-!!! note "This is the radius of the volume falloff distance in centimeters (cm). If the distance from the sound's origin is greater than the sound's radius, the sound will gradually decrease in volume over the falloff distance. The maximum range a sound can be heard is RADIUS + FALLOFF."
+Radius of the falloff distance. If the distance from the sounds's origin is greater than the sound's radium, the sound will gradually diminish over the falloff distance. The max range a sound can be heard at is **Radius** + **Falloff**.
```

Comment:
If the distance from the **sound's** origin is greater than the sound's **radius**